### PR TITLE
fix/ens-plan (PR 2)

### PR DESCRIPTION
**Pull request by kanej**
If a call includes the arg list (e.g. "setUniswap(boolean,address)") we need to escape the brackets and commas to form an allowed mermaid id.

I used a `window.diagram` local variable on the plan page to diagnose the issue and intend to retain it to help trouble shooting.

Fixes https://github.com/NomicFoundation/ignition/issues/351.

## Preview

image

### CODE REVIEW

**Comment by alcuadrado** [comment]
Option 1, definitely.

File: packages/ui/examples/ComplexModule.js [hunk 1]
```diff
@@ -31,7 +31,7 @@ const synthetixDefinition = defineModule("Synthetix", (m) => {
 
   const { router } = m.useModule(uniswap);
 
-  m.call(synthetixCore, "setUniswap", [router]);
+  m.call(synthetixCore, "setUniswap(boolean,address)", [false, router]);
```
Comment (alcuadrado) [inline packages/ui/examples/ComplexModule.js:34]: This should be just `setUniswap`. If this doesn't work, it's a bug.

File: packages/ui/test/to-mermaid.ts [hunk 2]
```diff
@@ -0,0 +1,269 @@
...
+
+          Module:Contract1["Deploy Contract1"]
+        end
+
+      classDef startModule stroke-width:4px`;
+
+    assertDiagram(moduleDefinition, expectedResult);
+  });
+
+  it.only("should render a module with a space in the name", () => {
```
Comment (alcuadrado) [inline packages/ui/test/to-mermaid.ts:36]: You forgot a `.only` here.

File: packages/ui/examples/ComplexModule.js (hunk 1 above)
Comment (kanej) [reply packages/ui/examples/ComplexModule.js:34]: You mean we should never have a user putting in the parameters in `setUniswap(boolean,address)`?

**Pull request merged**